notary-status.md

```diff
@@ -0,0 +1,183 @@
+
+Publishing and discovery of public keys should be easy for consumers to acquire, however, Notary v2 will not implicitly support a **T**rust **o**n **F**irst **U**se (TOFU) model.
+
+Key revocation must support air-gap environments, enabling users to validate keys when resources inside a network isolated environment are unable to reach public endpoints.
+
+### Designing Complex Systems Across Multiple Subject Matter Experts
+
+To deliver Notary v2, we recognized the need of experts from multiple backgrounds, experiences and skill sets. The various perspectives were needed to assure we learned from past efforts and learned from subject matter experts.
+
+As subject matter experts converged, we found it difficult for the various SMEs to understand other components of the end to end workflow. The typical Open Source model for authoring specs involves “writing it down”. Contributors Create a pull request on some markdown so all can review. However, we learned _The problem isn’t in the writing, it’s in the reading._
+
+To facilitate better communications across the skill sets, respecting everyone's time, we recognized the need to invest in models and prototypes. We followed the design patterns of other large, complex projects like Antoni Gaudí's design of [The Sagrada Famila](https://simple.wikipedia.org/wiki/Sagrada_Fam%C3%ADlia). The [sketch, prototype, build approach](https://stevelasker.blog/2020/07/31/sketch-prototype-build/) would enable the various experts to focus on their component, while understanding where they plug-into other components of the design.
+
+As a result, we identified the following stages of the Notary v2 effort:
+
+1. Define Requirements
+1. Build Prototypes
+1. Validate prototypes - learning, refining requirements, iterating prototypes
+1. Author a Notary v2 Spec
+
+## February 2021
+
+Since December 2019, teams have been meeting regularly, as captured in the [Notary v2 notes][nv2-notes]. Requirements and a focused prototype have been completed, enabling conversations to upstream projects.
+
+Updates on the **Areas of Focus** and **Stages** are provided below.
+
+### Stage Progress (Feb 2021)
+
+1. [**Define Requirements**](#define-requirements-(2021-02))
+1. [**Build Prototypes**](#build-prototypes-(2021-02))
+1. [**Validate Prototypes**](#validate-prototypes-(2021-02))
+1. [*Author a Notary v2 Spec*](#author-a-notary-v2-spec-(2021-02))
+
+#### Define Requirements (2021-02)
+
+[Baseline requirements](https://github.com/notaryproject/requirements#requirements) have been defined, enabling focused prototyping. Additional requirements are being discussed as PRs:
+
+- [Add ephemeral clients as a core scenario #25](https://github.com/notaryproject/requirements/issues/25)
+- [Should tags be signed in addition to digests? #43](https://github.com/notaryproject/requirements/issues/43)
+
+#### Build Prototypes (2021-02)
```

Comment:
Is this meant to imply we're going to be prototyping for two years?

Reply:
lol, it was intended to create a unique header/anchor 2012-February
Once I split the overview and specific status reports, I’ll change as we’ll only have a single heading per report.